packages/reva-app/src/pages/ProjectOrganisms.tsx

```diff
@@ -30,14 +29,22 @@ const Organisms: FC<PropsOrganisms> = ({
 
   if (!organisms) return <p>chargement des organismes...</p>;
 
+  const shuffledOrganisms = organisms
+    .map((organism) => ({ ...organism, sort: Math.random() }))
+    .sort((a, b) => a.sort - b.sort)
+    .map((organism) => {
+      const { sort, ...rest } = organism;
+      return rest;
+    });
+
```

Comment:
On préférera ne pas altérer les objets du tableau et ainsi éviter les 2 maps.

```suggestion
  const shuffledOrganisms = organisms
    .sort((a, b) => 0.5 - Math.random());
```